### ISSUE TITLE
Convert to Google Spreadsheet?

### ISSUE THREAD

**Issue by mikeal** [knode/node-meatspace#119]
So, @jlord just did some pretty cool stuff for NodeSchool using sheetsee. She was basically able to replace the pull request HTML stuff we were doing with a realtime Google spreadsheet.

This lowers that bar for contributions a bit and also allowed her to create a map visualization and a calendar visualization pretty easily. We might also be able to get more metadata about the events (meetup, workshop, nodeschool, conference, hackathon, etc).

What do people think?

**Comment by hackygolucky** [comment]
Sounds awesome @mikeal . Has she blogged about it yet? I know she's got pretty great documentation on sheetsee anyway, but specifically what you're talking about from nodeschool?

**Comment by mikeal** [comment]
Nope, but we could probably just steal the code from the node school repo :)

https://github.com/nodeschool/nodeschool.github.io

**Comment by hackygolucky** [comment]
I'll take a peek at it tonight :)

**Comment by jlord** [comment]
Wee! I'm happy to help out, let me know!

**Comment by hackygolucky** [comment]
Thanks @jlord! I'm sure you'll be hearing from me :)

**Comment by gergelyke** [comment]
Any update on this one? In the current form the repos seems dead :(

**Comment by jlord** [comment]
If someone creates a spreadsheet (assuming one of you'd want to own it, but give me edit access) and posts the URL here I think I can hack on it this weekend.
